src/components/atoms/Ticker/ticker.css

```diff
@@ -0,0 +1,24 @@
+.ticker42hiddenPiece {
+  visibility: hidden;
+}
+
+.ticker42Rotator {
+  transition: transform 0.5s;
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+}
+
+.ticker42Tick {
+  overflow: hidden;
+  display: inline-block;
+  position: relative;
+}
+
+.ticker42Piece {
+  position: absolute;
+  left: 0;
+  z-index: 10;
```

Comment:
we can discuss this.

Reply:
discussed already and changed in the file.